testsupport/tiers/nstemplateset.go

```diff
@@ -46,10 +46,20 @@ func VerifyNSTemplateSet(t *testing.T, hostAwait *wait.HostAwaitility, memberAwa
 		}
 		spaceRoles := map[string][]string{}
 		for _, r := range nsTmplSet.Spec.SpaceRoles {
-			tmpl, err := hostAwait.WaitForTierTemplate(t, r.TemplateRef)
-			require.NoError(t, err)
-			t.Logf("space role template '%s' for usernames '%v' in NSTemplateSet '%s'", tmpl.GetName(), r.Usernames, nsTmplSet.Name)
-			spaceRoles[tmpl.Spec.Type] = r.Usernames
+			if nsTmplSet.Spec.TierName == "ttr-go-template" {
```

Comment:
Is there a more robust way of determining whether the set is backed by a go-template or OS template? Distinguishing by name seems quite brittle to me...

Reply:
so for nstemplateset i couldn't find a distinguisher, may be i am missing something.
i have tried to update in other places
though this won't be required as soon as we remove the openshift template